code/game/gamemodes/wizard/wizloadouts.dm

```diff
@@ -37,21 +37,26 @@
 /datum/spellbook_entry/loadout/mimewiz
 	name = "Silencio"
 	desc = "...<br><br> \
-		</i>Provides Finger Gun and Invisible Greater Wall manuals, Mime Robes, a Cane and Duct Tape, Ethereal Jaunt, Blink, Teleport, Mime Malaise, Knock, and Stop Time.<i>"
+		</i>Provides Finger Gun and Invisible Greater Wall, Mime Robes, a Cane and Duct Tape, Ethereal Jaunt, Teleport, Mime Malaise and Knock.<i>"
 	log_name = "SHH"
-	items_path = list(/obj/item/spellbook/oneuse/mime/fingergun, /obj/item/spellbook/oneuse/mime/greaterwall, /obj/item/clothing/suit/wizrobe/mime, /obj/item/clothing/head/wizard/mime, \
+	items_path = list(/obj/item/clothing/suit/wizrobe/mime, /obj/item/clothing/head/wizard/mime, \
 		/obj/item/clothing/mask/gas/mime/wizard, /obj/item/clothing/shoes/sandal/marisa, /obj/item/cane, /obj/item/stack/tape_roll)
-	spells_path = list(/obj/effect/proc_holder/spell/targeted/ethereal_jaunt, /obj/effect/proc_holder/spell/targeted/turf_teleport/blink, /obj/effect/proc_holder/spell/targeted/area_teleport/teleport, \
-		/obj/effect/proc_holder/spell/targeted/touch/mime_malaise, /obj/effect/proc_holder/spell/aoe_turf/knock, /obj/effect/proc_holder/spell/aoe_turf/conjure/timestop)
+	spells_path = list(/obj/effect/proc_holder/spell/targeted/ethereal_jaunt, /obj/effect/proc_holder/spell/targeted/area_teleport/teleport, \
+		/obj/effect/proc_holder/spell/targeted/touch/mime_malaise, /obj/effect/proc_holder/spell/aoe_turf/knock, \
+		/obj/effect/proc_holder/spell/targeted/mime/speak/wizard, /obj/effect/proc_holder/spell/targeted/forcewall/mime, \
+		/obj/effect/proc_holder/spell/targeted/mime/fingergun)
```

Comment:
All looks good code-wise aside from the obligatory "Don't need a backslash after a comma in a list" issue.